javascript/javascript1/week4/homework.js

```diff
@@ -0,0 +1,120 @@
+//Voice Assistant
+var user = {
+    "name": "",
+    "toDoList": []
+};
+function getReply(command) {
+    let voiceCom = command;
```

Comment:
You can use `command` directly. You don't need to reassign it to `let voiceCom = command;`
```suggestion
function getReply(voiceCom) {
```

Reply:
But then I need again the let voiceCom = command to put the command in lower case.

Reply:
True, you can combine `let command = voiceCom` and `voiceCom.toLowerCase()`.